File: stock-serveur-v2/src/main/java/cm/belrose/stockserveur/model/LigneCommandeFournisseur.java
package cm.belrose.stockserveur.model;

import cm.belrose.stockserveur.config.audit.Auditable;
import lombok.*;
import org.hibernate.envers.NotAudited;

import javax.persistence.*;
import java.io.Serializable;
import java.math.BigDecimal;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "lignecommandefournisseurs")
public class LigneCommandeFournisseur extends Auditable<String> implements Serializable {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private BigDecimal quantite;
    private BigDecimal prixUnitaire;
    private Long entrepriseId;//Ce champs est juste mis pour simplifier les choses
    @NotAudited
    @ManyToOne
    private Article article;
    @ManyToOne
    private CommandeFournisseur commandeFournisseur;

}

File: stock-client-v2/src/gs-api/src/models/ligne-commande-fournisseur.ts
/* tslint:disable */
import { Article } from './article';
import { CommandeFournisseur } from './commande-fournisseur';
export interface LigneCommandeFournisseur {
  createdBy?: string;
  updatedBy?: string;
  createdDate?: string;
  updateDate?: string;
  id?: number;
  quantite?: number;
  prixUnitaire?: number;
  entrepriseId?: number;
  article?: Article;
  commandeFournisseur?: CommandeFournisseur;
}
